driver/flambda_backend_flags.ml

```diff
@@ -28,6 +28,12 @@ let alloc_check = ref false             (* -alloc-check *)
 let dump_checkmach = ref false          (* -dcheckmach *)
 
 let disable_poll_insertion = ref false  (* -disable-poll-insertion *)
+let allow_long_frames = ref true        (* -no-long-frames *)
+(* Keep the value of [max_long_frames_threshold] in sync with LONG_FRAME_MARKER
+   in ocaml/runtime/roots_nat.c *)
+let max_long_frames_threshold = 0x7FFF
+let long_frames_threshold = ref max_long_frames_threshold (* -long-frames-debug-threshold n *)
```

Comment:
This comment doesn't match the flag name

Reply:
done